user-guide/Getting_started/DaaS/Order_DataMiner_credits.md

```diff
@@ -0,0 +1,50 @@
+---
+uid: Order_DataMiner_credits
+---
+
+# Ordering DataMiner credits
+
+To create and maintain a DaaS instance, your [organization](xref:Pricing_Usage_based_service#organization) must have sufficient DataMiner credits. To order DataMiner credits for the first time, follow the procedure below.
+
+> [!NOTE]
+> If you have already created a SaaS subscription as detailed below, and you want to order **additional credits**, you do not have to create a new SaaS subscription. Instead, go to your SaaS subscription in the Azure portal, and click *Open SaaS Account on publisher's site* to be redirected to the *Purchase credits* page.
+
+1. Go to the [Azure Marketplace](https://azuremarketplace.microsoft.com/) and search for DataMiner.
+
+1. Click *Get It Now* to be redirected to the Azure portal.
+
+   > [!IMPORTANT]
+   > Make sure you have selected the plan *DataMiner credits*.
```

Comment:
My original feedback still applies here: it would be better to add this as a separate step (with screenshot), before the "Get It Now" step. Otherwise, you risk that a user clicks "Get It Now" before they see that they should have had "DataMiner credits" selected.